From 5848a5f2939994a60d0004876f0c5d7e8240d28b Mon Sep 17 00:00:00 2001
From: Naoto Yamaguchi <naoto.yamaguchi@aisin.co.jp>
Date: Sat, 13 Aug 2022 14:21:07 +0900
Subject: [PATCH] Fix build issue on Yocto dunfell

---
 Makefile | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Makefile b/Makefile
index b4179da8e8..03019a3efc 100644
--- a/Makefile
+++ b/Makefile
@@ -1699,6 +1699,13 @@ endif
 	$(build)=$(build-dir) $(@:.ko=.o)
 	$(Q)$(MAKE) -f $(srctree)/scripts/Makefile.modpost
 
+quiet_cmd_genenv = GENENV $@
+cmd_genenv = $(OBJCOPY) --dump-section .rodata.default_environment=$@ env/common.o; \
+	sed --in-place -e 's/\x00/\x0A/g' $@
+
+u-boot-initial-env: u-boot.bin
+	$(call if_changed,genenv)
+
 # FIXME Should go into a make.lib or something
 # ===========================================================================
 
-- 
2.25.1

