From f3fcd4c3a20b243251e656cff981ae747f9d6db0 Mon Sep 17 00:00:00 2001
From: Naoto Yamaguchi <naoto.yamaguchi@aisin.co.jp>
Date: Sun, 14 Aug 2022 03:01:51 +0900
Subject: [PATCH] Set bootargs for SD boot

---
 arch/arm/mach-rockchip/boot_rkimg.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-rockchip/boot_rkimg.c b/arch/arm/mach-rockchip/boot_rkimg.c
index 96e051663e..b15411c8ec 100644
--- a/arch/arm/mach-rockchip/boot_rkimg.c
+++ b/arch/arm/mach-rockchip/boot_rkimg.c
@@ -47,6 +47,8 @@ static void boot_devtype_init(void)
 	if (done)
 		return;
 
+	env_set("bootargs", "console=ttyFIQ0 root=/dev/mmcblk1p4 rw ");
+
 	/* High priority: get bootdev from atags */
 #ifdef CONFIG_ROCKCHIP_PRELOADER_ATAGS
 	ret = param_parse_bootdev(&devtype, &devnum);
-- 
2.25.1

